### PR TITLE
Modify multiple while

### PR DESCRIPTION
**Pull request by lukyanoffpashok**
#447

### CODE REVIEW

File: aibolit/patterns/multiple_while/multiple_while.py [hunk 1]
```diff
@@ -16,9 +15,17 @@ def value(self, filename: str):
...
 
+        tree = AST(build_ast(filename))
         res = []
-        for _, method_node in build_ast(filename).filter(javalang.tree.MethodDeclaration):
-            if len(list(method_node.filter(javalang.tree.WhileStatement))) > 1:
-                res.append(method_node.position.line)
+        nodes = tree.nodes_by_type(ASTNodeType.METHOD_DECLARATION)
+        for node in nodes:
+            cur_line = tree.get_attr(node, 'source_code_line')
+            while_cyclo = tree.all_children_with_type(node, ASTNodeType.WHILE_STATEMENT)
```
Comment (aravij) [inline aibolit/patterns/multiple_while/multiple_while.py:23]: Maybe cycle?

File: aibolit/patterns/multiple_while/multiple_while.py [hunk 2]
```diff
@@ -16,9 +15,17 @@ def value(self, filename: str):
...
+            count = 0
+            for cyclo in while_cyclo:
```
Comment (aravij) [inline aibolit/patterns/multiple_while/multiple_while.py:25]: It is better convert generator to list and check its length.
I am currently consider using generators in AST was mine mistake. We might git rid of them soon.

File: aibolit/patterns/multiple_while/multiple_while.py (hunk 1 above)
Comment (lukyanoffpashok) [reply aibolit/patterns/multiple_while/multiple_while.py]: fix

File: aibolit/patterns/multiple_while/multiple_while.py (hunk 2 above)
Comment (lukyanoffpashok) [reply aibolit/patterns/multiple_while/multiple_while.py]: ok

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#448` is not assigned, can't get performer

**Pull request merged**